inbound.go

```diff
@@ -42,12 +46,19 @@ func (c *Connection) handleCallReq(frame *Frame) bool {
 		c.SendSystemError(frame.Header.ID, nil, ErrChannelClosed)
 		return true
 	case connectionWaitingToRecvInitReq, connectionWaitingToSendInitReq, connectionWaitingToRecvInitRes:
-		c.SendSystemError(frame.Header.ID, nil, NewSystemError(ErrCodeDeclined, "connection not ready"))
+		c.SendSystemError(frame.Header.ID, nil, errNotReady)
 		return true
 	default:
 		panic(fmt.Errorf("unknown connection state for call req: %v", state))
 	}
 
+	return c.handleInboundCall(frame)
+}
+
+// handleInboundCall handles an incoming call request, registering a message
+// exchange to receive further fragments for that call, and dispatching it in
+// another goroutine
+func (c *Connection) handleInboundCall(frame *Frame) bool {
```

Comment:
Breaking this function up results in compiler errors:

```
Testing packages:
# github.com/uber/tchannel-go
./inbound.go:94: undefined: now
./inbound.go:113: undefined: now
FAIL	github.com/uber/tchannel-go/stats [build failed]
```

Looks like this helper will need to take `now` as a param.